test failure looks real
```
ERROR: corehq.apps.users.tests.test_migrate_roles_to_sql:UserRoleCouchToSqlTests.test_diff_assignable_by_couch_None
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/users/tests/test_migrate_roles_to_sql.py", line 144, in test_diff_assignable_by_couch_None
    self.assertListEqual(Command.get_filtered_diffs(couch.to_json(), sql), [
  File "/mnt/commcare-hq-ro/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py", line 66, in get_filtered_diffs
    diffs = cls.diff_couch_and_sql(couch, sql)
  File "/mnt/commcare-hq-ro/corehq/apps/users/management/commands/populate_user_role.py", line 55, in diff_couch_and_sql
    sorted(couch_assignable_by),
TypeError: 'NoneType' object is not iterable
```

--- a/corehq/apps/users/management/commands/populate_user_role.py
+++ b/corehq/apps/users/management/commands/populate_user_role.py
@@ -48,11 +48,11 @@ class Command(PopulateSQLCommand):
                 name_prefix=f"permissions.{name}"
             ))
 
-        couch_assignable_by = couch.get("assignable_by", [])
+        couch_assignable_by = couch.get("assignable_by")
         sql_assignable_by = list(sql.roleassignableby_set.values_list('assignable_by_role__couch_id', flat=True))
         diffs.extend(cls.diff_lists(
             "assignable_by",
-            sorted(couch_assignable_by),
+            sorted(couch_assignable_by) if couch_assignable_by else [],
             sorted(sql_assignable_by),
         ))
         return diffs

